### DISCUSSION TITLE
Github Enterprise and self-hosted runners with Dependabot

### DISCUSSION THREAD

**Discussion by inverse** [community/community#33826] [Enterprise]
I saw that this should work on the [Enterprise Server Plan](https://docs.github.com/en/enterprise-server@3.3/admin/github-actions/enabling-github-actions-for-github-enterprise-server/managing-self-hosted-runners-for-dependabot-updates) by tagging the runner with the `dependabot` label.

But I was wondering if there was a way to get this working on the managed Enterprise tier plans? 

Since our artefact repositories are not exposed to the outside world we are unable to work with Dependabot in it's managed form.

**Comment by sgandon** [comment]
I am having the same question I guess. We are using a GitHub Enterprise Cloud plan and are using self-hosted runners for security reason.
I there a way to run dependabot on our self-hosted runners so that it can access our private registries ?

**Comment by jorsmatthys** [comment]
We ran into the same limitation while testing dependabot for our enterprise.
For now we use dependabot, but without generating pull requests for our internal dependencies stored in our self-hosted Nexus. 

I saw that there are alternative tools that we could use for this (renovate etc.), but dependabot is very easy to configure so we would prefer seeing this feature added to dependabot. I imagine this will make the feature to specify your own repositories/registries in dependabot.yml useful to a lot more people.

**Comment by kuhnroyal** [comment]
There is a private beta where this is already possible in GHEC. And here is the roadmap ticket, currently planned for Q2 https://github.com/github/roadmap/issues/929
